timesketch/frontend/src/components/Explore/EventListRowDetail.vue

```diff
@@ -157,6 +167,9 @@ export default {
       }
       return regexes
     },
+    getEventLink: function () {
+      return encodeURI(`${window.location.origin}/sketch/${this.$store.state.sketch.id}/explore?q=_id:"${this.event._id}"&timeline=${this.event._source.__ts_timeline_id}`);
+    },
```

Comment:
It would be preferable to leverage the Vue router to generate this link

Reply:
@anttitikkanen would you mind to make that with Vue? then this should be good to be merged.